common/Printer.java

```diff
@@ -87,9 +86,8 @@ private String conceptMapDisplayString(ConceptMap conceptMap, GraknTransaction t
         StringBuilder sb = new StringBuilder();
         sb.append("{ ");
         for (Map.Entry<String, Concept> entry : conceptMap.map().entrySet()) {
-            Reference.Name variable = Reference.name(entry.getKey());
             Concept concept = entry.getValue();
-            sb.append(variable.syntax());
+            sb.append(GraqlToken.Char.$).append(entry.getKey());
             sb.append(" ");
```

Comment:
we don't purely return `Reference.name` anymore, sometimes its a `Reference.anonymous`, but with an extra ID (because it comes from the server). To work around this we just render the name directly and prepend `$` in the printer